src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs

```diff
@@ -190,6 +190,61 @@ static void Main()
         </Document>
         <Document>// &lt;autogenerated /&gt;
 
+class Program2
+{
+}
+        </Document>
+    </Project>
+</Workspace>");
+        }
+
+        [WorkItem(32792, "https://github.com/dotnet/roslyn/issues/32792")]
```

Comment:
Could you add a test with multi-line comment banner please

Reply:
done.